cmd/minikube/cmd/start.go

```diff
@@ -222,6 +222,8 @@ func runStart(cmd *cobra.Command, args []string) {
 	host, preexisting := startHost(m, config.MachineConfig)
 
 	ip := validateNetwork(host)
+	// Makes minikube node ip to bypass http(s) proxy. since it is local traffic.
+	updateNoProxy(ip)
```

Comment:
This should be inserted somewhere further up (main.go), so that "minikube dashboard' and "minikube status" work as well with misconfigured proxies.